pkg/steps/clusterinstall/template.go

```diff
@@ -312,9 +283,7 @@ objects:
           export KUBE_SSH_USER=core
           mkdir -p ~/.ssh
           cp /tmp/cluster/ssh-privatekey ~/.ssh/google_compute_engine || true
-          # TODO: make openshift-tests auto-discover this from cluster config
-          REGION="$(oc get -o jsonpath='{.status.platformStatus.gcp.region}' infrastructure cluster)"
-          export TEST_PROVIDER="{\"type\":\"gce\",\"region\":\"${REGION}\",\"multizone\": true,\"multimaster\":true,\"projectid\":\"openshift-gce-devel-ci\"}"
+          export TEST_PROVIDER='{"type":"gce","region":"us-east1","multizone": true,"multimaster":true,"projectid":"openshift-gce-devel-ci"}'
```

Comment:
this is backsliding vs openshift/ci-tools#1527 , because I haven't been able to convince anyone to review openshift/release#14324, and haven't bothered pushing the change to other sections of openshift/release with that PR still in flight.